docs/tech/application-ACL.md

```diff
@@ -118,50 +118,50 @@ message Transaction {
 
     bytes payload;
     bytes metadata;
-    
-    ...    
+
+    ... 
 }
 ```
 
-Another way to achieve this is to have the payload containing the metadata itself.  
+Another way to achieve this is to have the payload containing the metadata itself.
 
 ### Validators
 
-To assist chaincode execution, the validators provide the chaincode additional information, like the metadata and the binding.  
- 
+To assist chaincode execution, the validators provide the chaincode additional information, like the metadata and the binding.
+
 ## Application-level access control
- 
+
 ### Deploy Transaction
- 
-Alice has full control on the deployment transaction's metadata. 
-In particular, the metadata can be used to store a list of ACLs (one per function), or a list of roles. 
-To define each of these lists/roles, Alice can use any TCerts/ECerts of the users who have been assigned 
-that privilege or role. The latter is done offline. 
 
-Now, Alice requires that to invoke the *hello* function, a certain message *M* has to be authenticated by an authorized invoker (Bob, in our case). 
+Alice has full control on the deployment transaction's metadata.
+In particular, the metadata can be used to store a list of ACLs (one per function), or a list of roles.
+To define each of these lists/roles, Alice can use any TCerts/ECerts of the users who have been assigned
```

Comment:
To define each of these lists/roles, Alice can use any TCerts/ECerts of the users who have been assigned that privilege or role. 

to 

To define each of these lists/roles, Alice can use any TCerts/ECerts of the users who have been 
**granted that (access control) privilege or have been assigned that role**